kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -317,6 +432,15 @@ private static Map<String, Variant<?>> createWEPSettings(NetworkProperties props
         return settings;
     }
 
+    private static Map<String, Variant<?>> createWPA2WPA3EnterpriseSettings(NetworkProperties props, String deviceId,
```

Comment:
To provide some additional consistency/tidy things up consider moving `createWPA2WPA3EnterpriseSettings` body after `createWPAWPA2Settings`. This follows the call order in the switch above.